### DISCUSSION TITLE
https://cucoders.dev/publicaciones/kader1303/observable-pattern-una-alternativa-al-usecontext-en-react-y-react-native/

### DISCUSSION THREAD

**Discussion by giscus** [CuCodersCommunity/cucoderscommunity.github.io#57] [Page Comments]
# https://cucoders.dev/publicaciones/kader1303/observable-pattern-una-alternativa-al-usecontext-en-react-y-react-native/

Comparación entre dos métodos para mantener la sincronización de datos en React: el Observable Pattern y el useContext. Ejemplos de cómo implementarlos, ventajas, desventajas y principales diferencias entre ambos.

https://cucoders.dev/publicaciones/kader1303/observable-pattern-una-alternativa-al-usecontext-en-react-y-react-native/

**Comment by manuelernestog** [comment]
Muy bueno el artículo bro 👏
